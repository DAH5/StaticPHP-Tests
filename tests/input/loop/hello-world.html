---
title: Hello, world!
description: This is a hello world file.
---
<p>--- metadata.title ---</p>
<p>--- metadata.description ---</p>