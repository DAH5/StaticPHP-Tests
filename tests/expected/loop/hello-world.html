<p>Hello, world!</p>
<p>This is a hello world file.</p>